Makefile

```diff
@@ -1,20 +1,29 @@
 PREFIX?=/usr/local
 
+TEMPORARY_FOLDER=./tmp_portable_licenseplist
+
 build:
 	swift build -c release -Xswiftc -static-stdlib
- 
-test: 
+
+test:
 	swift test
 
 lint:
 	swiftlint
- 
+
 clean:
 	swift package clean
 
 xcode:
 	swift package generate-xcodeproj
-	
+
 install: build
 	mkdir -p "$(PREFIX)/bin"
 	cp -f ".build/release/LicensePlist" "$(PREFIX)/bin/license-plist"
```

Comment:
The executable binary is renamed to chain case from camel case here when installed via Makefile or Homebrew.
It is better to also apply it `cp -f ".build/release/LicensePlist" "$(TEMPORARY_FOLDER)"`, isn't it?
(I don't understand CocoaPods distribution well.)